client.go

```diff
@@ -169,8 +175,12 @@ func (c *Client) handleLine(line string) {
 			Tags:   message.Tags,
 			Text:   message.Text,
 		}
-
+		
 		switch message.Type {
+		case WHISPER:
```

Comment:
Move this under the PRIVMSG, I prefer to sort these by occurence a bit and a PRIVMSG is going to be a lot more common usally.

Reply:
Done!